rules/kubelet-hostname-override/raw.rego

```diff
@@ -0,0 +1,24 @@
+package armo_builtins
+
+import future.keywords.in
+
+# --hostname-override argument is not present
+deny[msga] {
+	kubelet_info := input[_]
+	kubelet_info.kind == "KubeletInfo"
+	kubelet_info.apiVersion == "hostdata.kubescape.cloud/v1beta0"
+	command := kubelet_info.data.cmdLine
+
+	contains(command, "--hostname-override")
```

Comment:
`--config`?

Reply:
This setting is not configurable via the Kubelet config file.